### DISCUSSION TITLE
mobile: iPhone11 , ios version: 13.6.1, visit mdn docs had page blank

### DISCUSSION THREAD

**Discussion by liaolongdong** [mdn/mdn-community#432] [Content]
Device Information：mobile: iPhone11 , ios version: 13.6.1
I use an Apple mobile phone to access mdn docs page and it is blank, why got this error?

**Comment by bsmth** [comment]
Hi @liaolongdong - this sounds more like a platform issue - whereas the discussions here are not for bug reports. If you would like, the correct place to report an issue about this problem is [in the Yari repository](https://github.com/mdn/yari/issues/new/choose).

I'm going to close this one for now. Thank you :)

**Reply by liaolongdong** [reply]
OK，thinks
